src/components/ProfilePage/QuestCarousel/QuestCarousel.tsx

```diff
@@ -29,10 +29,13 @@ export const QuestCarousel = ({ quests, loading }: QuestCarouselProps) => {
                       key={`ongoing-mission-${index}`}
                       active={true}
                       title={quest?.attributes.Title}
-                      image={new URL(
-                        quest.attributes.Image?.data?.attributes?.url,
-                        url.origin,
-                      ).toString()}
+                      image={
+                        quest.attributes.Image?.data?.attributes?.url &&
```

Comment:
Do it at the component level instead of the image field please otherwise it is not solving the issue